tests/test-parse-multiline.js

```diff
@@ -23,9 +23,9 @@ t.equal(parsed.DOUBLE_QUOTES_SPACED, '    double quotes    ', 'respects surround
 
 t.equal(parsed.EXPAND_NEWLINES, 'expand\nnew\nlines', 'expands newlines but only if double quoted')
 
-t.equal(parsed.DONT_EXPAND_UNQUOTED, 'dontexpand\\nnewlines', 'expands newlines but only if double quoted')
+t.equal(parsed.DONT_EXPAND_UNQUOTED, 'dontexpandnnewlines', 'expands newlines but only if double quoted')
```

Comment:
i have mixed feelings on this behavior. looks like the rubygem has done this since inception, but it doesn't follow the principle of least surprise to me. the `\n` disappearing would frustrate me.

Reply:
this behavior goes back to `0.0.3`. https://github.com/motdotla/dotenv/commit/599144a3d2e22f6539a9429d409f1723769736b6

i'm open to arguments for doing this, but it will definitely be a potentially significant breaking change if we do